composeApp/src/commonMain/kotlin/sp/bvantur/tasky/event/ui/components/TaskyVisitorsSection.kt

```diff
@@ -17,33 +25,110 @@ import androidx.compose.ui.Modifier
+                color = MaterialTheme.colorScheme.primary
+            )
+
+            IconButton(
+                modifier = Modifier.padding(start = 16.dp),
+                onClick = onClick
+            ) {
+                Icon(
+                    modifier = Modifier.size(35.dp).clip(RoundedCornerShape(5.dp))
+                        .background(MaterialTheme.colorScheme.tertiary).padding(5.dp),
+                    imageVector = Icons.Default.Add,
+                    contentDescription = stringResource(Res.string.add_visitors),
+                    tint = MaterialTheme.colorScheme.onTertiary
+                )
+            }
+        }
 
-        IconButton(
-            modifier = Modifier.padding(start = 18.dp),
-            onClick = onClick
+        LazyColumn(modifier = Modifier.fillMaxWidth()) {
+            items(attendees.size) { index ->
+                AttendeeItem(
+                    attendee = attendees[index],
+                    onDeleteAttendee = onDeleteAttendee
+                )
+            }
+        }
+    }
+}
+
+@Composable
+private fun AttendeeItem(modifier: Modifier = Modifier, attendee: Attendee, onDeleteAttendee: (Attendee) -> Unit) {
+    Card(
+        modifier = modifier.fillMaxWidth().padding(all = 16.dp),
+        colors = CardDefaults.cardColors(
+            containerColor = MaterialTheme.colorScheme.tertiary
+        )
+    ) {
+        Row(
+            verticalAlignment = Alignment.CenterVertically,
+            modifier = Modifier.padding(start = 16.dp)
         ) {
-            Icon(
-                modifier = Modifier.size(35.dp).clip(RoundedCornerShape(5.dp))
-                    .background(MaterialTheme.colorScheme.tertiary).padding(5.dp),
-                imageVector = Icons.Default.Add,
-                contentDescription = stringResource(Res.string.add_visitors),
-                tint = MaterialTheme.colorScheme.onTertiary
+            InitialsCircle(
+                attendee.name,
+                modifier = Modifier.padding(vertical = 6.dp)
+            )
+
+            Text(
+                modifier = Modifier.weight(1f).padding(horizontal = 16.dp),
+                text = attendee.name,
+                style = MaterialTheme.typography.labelSmall,
+                color = MaterialTheme.colorScheme.eventChoreTitleType
             )
+
+            IconButton(onClick = { onDeleteAttendee(attendee) }) {
+                Icon(
+                    imageVector = Icons.Sharp.Delete,
+                    contentDescription = stringResource(Res.string.password_visibility_icon),
+                    tint = MaterialTheme.colorScheme.eventChoreTitleType
+                )
+            }
         }
     }
 }
+
+@Composable
+private fun InitialsCircle(value: String, modifier: Modifier = Modifier) {
+    val initials = value.split(" ")
+        .filter { it.isNotBlank() }
+        .take(2)
+        .joinToString("") { it.first().uppercase() }
```

Comment:
This doesn't work for middle names. For `Philipp Middle Lackner`, it would create the initials `PM` instead of `PL`.
Also, just some food for thought: What if the user enters weird characters or emojis? This is not explicitly in the requirements how this should be handled, but maybe there should be some logic to avoid having emojis in the initials 😄 When writing such algorithms, I try to put myself into a perspective of someone who would get money for breaking this code with valid inputs. Then you suddenly don't only think of the happy path

Reply:
I will add TODO and work on such edge cases after our mentorship program is done. I don't want to waste too much time covering the edge cases of the screens. I want to move forward fast to cover other major topics. Still, appreciate you writing such comments. Thank you :)